docs/modules/ROOT/pages/commands.adoc

```diff
@@ -0,0 +1,437 @@
+:starknet-devnet: link:https://github.com/Shard-Labs/starknet-devnet/[starknet-devnet]
+:cairo-lang: link:https://github.com/starkware-libs/cairo-lang[cairo-lang]
+:imports: link:https://starknet.io/docs/how_cairo_works/imports.html?highlight=cairo%20path#import-search-paths[Import search paths]
+:status: link:#status[status]
+
+== CLI Reference
+
+Here you can find a complete reference of the core commands Nile provides by default.
+
+[.contract-index]
+.Core Commands
+--
+* xref:#node[`++node++`]
+* xref:#compile[`++compile++`]
+* xref:#setup[`++setup++`]
+* xref:#declare[`++declare++`]
+* xref:#deploy[`++deploy++`]
+* xref:#call[`++call++`]
+* xref:#send[`++send++`]
+* xref:#counterfactual-address[`++counterfactual-address++`]
+* xref:#status[`++status++`]
+* xref:#debug[`++debug++`]
+* xref:#get-accounts[`++get-accounts++`]
+* xref:#get-nonce[`++get-nonce++`]
+* xref:#get-balance[`++get-balance++`]
+* xref:#run[`++run++`]
+* xref:#clean[`++clean++`]
+* xref:#init[`++init++`]
+* xref:#version[`++version++`]
```

Comment:
We could do something like what is presented in the screenshot below (**I don't particularly like it more than what we have**), but we have the constraint that the UI components are generated from rules specified in the documentation repository, and I can't find a better approach to this with what is implemented there (I honestly don't want to update UI components on that repo without a predefined design).

<img width="795" alt="Screenshot 2022-12-27 at 16 16 49" src="https://user-images.githubusercontent.com/26070224/209686405-8cbd1308-9597-4a3c-90cf-1e4fb76696cc.png">

Reply:
i like the formatting. let's work a bit more around the grouping. not sure `call`, `status`, and `debug` are just utils.

Reply:
This can be a starting point:

**Write Operations**

- setup
- declare
- deploy
- send

**Read Operations**

- call
- get-nonce
- get-balance

**Transaction Status**

- status
- debug

**Utils**

- get-accounts
- counterfactual-address

**Project Tasks**
- init
- node
- compile
- run
- clean
- version

Reply:
i like that. it helps understand that write ops = account ops

Reply:
Are you suggesting changing "Write Operations" to "Account Operations"? What about the other groups?

Reply:
i wasn't, just pointing out that write operations are the ones that require accounts. i think the grouping is good but not that convinced about the names.

maybe:
* transactions ("write" is possibly implied, maybe status ones there too?)
* queries
* project management
* utils

Reply:
That sounds better. Format updated.